app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,7 +1,7 @@
 /* Main navbar */
 .navbar-wagon {
   box-shadow: 0 1px 5px 0 rgba(0,0,0,0.07),0 1px 0 0 rgba(0,0,0,0.03);
-  background: white;
+  background: #89a4c7;
```

Comment:
You should use a scss variable

Reply:
ok mais c'était juste pour qu'on voit le logo

Reply:
Yes pas de souci, ça donnera la même chose, mais c'est juste pour que tu aies tes couleurs dans des variables scss et pouvoir les réutiliser facilement